### PR TITLE
Fix local setup

### LINKED ISSUE #37: Instructions for local setup are out of date
I think the changes while switching to github actions broke something

### PR DESCRIPTION
**Pull request by BlackYps**
Closes #37 
I also made the container restart automatically so you don't have to set it up again every time you want to run the unit tests.
However, waiting for the pid file doesn't seem to work. I don't really know why.

### CODE REVIEW

File: scripts/local_rabbitmq.sh [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+RABBITMQ_PID_FILE=/var/lib/rabbitmq/pid
+RABBITMQ_DEFAULT_USER=admin
+RABBITMQ_DEFAULT_PASS=banana
+RABBITMQ_DEFAULT_VHOST=/
+MAX_WAIT_SECONDS=30
+RABBITMQ_LEAGUE_SERVICE_USER=faf-league-service
+RABBITMQ_LEAGUE_SERVICE_PASS=banana
+RABBITMQ_LEAGUE_SERVICE_VHOST=/faf-lobby
+
+docker run -d -p 5672:5672 --restart unless-stopped --name faf-rabbitmq rabbitmq:3.8.2-management-alpine
```
Comment (Askaholic) [inline scripts/local_rabbitmq.sh]: Is it necessary to specify the bugfix version?
```suggestion
docker run -d -p 5672:5672 --restart unless-stopped --name faf-rabbitmq rabbitmq:3.8-management-alpine
```

Comment (BlackYps) [reply scripts/local_rabbitmq.sh]: I guess it is not needed

Comment (BlackYps) [reply scripts/local_rabbitmq.sh]: I went ahead and also removed the minor version, because test.yml also uses the latest minor version

Comment (Askaholic) [reply scripts/local_rabbitmq.sh]: Just be careful because I had issues with 3.9 on the lobby server so I had to pin it at 3.8.

**Review by Askaholic — APPROVED** [review]

File: scripts/local_rabbitmq.sh (hunk 1 above)
Comment (BlackYps) [reply scripts/local_rabbitmq.sh]: yeah, I should probably use 3.8 like also defined in faf stack. no use if it works locally but breaks when it is actually deployed :D

**Pull request merged**